tests/build/jvm-build.go

```diff
@@ -247,103 +230,103 @@ var _ = framework.JVMBuildSuiteDescribe("JVM Build Service E2E tests", func() {
 				return true
 			}, timeout, interval).Should(BeTrue(), "timed out when waiting for the pipeline run to complete")
 		})
+		// Blocked by https://issues.redhat.com/browse/PLNSRVCE-519
```

Comment:
I think it will be nicer to skip these `It` containers and put the issue into the skip message (https://onsi.github.io/ginkgo/#skipping-specs)

Reply:
good point, I will fix it in a follow-up PR